EIPS/eip-Claimable_Token.md

```diff
@@ -0,0 +1,162 @@
+---
+eip: 2871
+title: tandard for BlockChain Mail Stamp
+author: Wansheng Li huhulws@gmail.com
+status: Draft
+type: Standards Track
+category: ERC
+created: 2020-8-10
+requires: 20
```

Comment:
Missing `discussion-to` header field.

Reply:
Thank you for commenting! The reason why build claim function into token is that we want add modifier 'onlyIssuer', which means only the token issuer can claim to avoid double spending problem. If there are mutiply claimers, we should sync micropayment messages between them. And in our projects, the token issuers are distributed servers/miners that provide service for user to choice. But I think you are right, we could make it separete, and add a mapping between token address and beneficiary. This strategy is better than traditional payment because it's decentralized and no bank account involved. I should write more about this later. Thanks again!